ee/ucp/interlock/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Layer 7 routing overview
+description: Learn how to route layer 7 traffic to your Swarm services
+keywords: routing, UCP, interlock, load balancing
+---
+
+Application-layer (Layer 7) routing is the application routing and load balancing (ingress routing) system included with Docker Enterprise for Swarm orchestration. Interlock architecture takes advantage of the underlying Swarm components to provide scalable Layer 7 routing and Layer 4 VIP mode functionality.
+
+Interlock is specific to the Swarm orchestrator. If you're trying to route traffic to your Kubernetes applications, check [layer 7 routing with Kubernetes.](../kubernetes/layer-7-routing.md)
+
+Interlock uses the Docker Remote API to automatically configure extensions such as NGINX or HAProxy for application traffic. Interlock is designed for:
+
+- Full integration with Docker (Swarm, Services, Secrets, Configs)
+- Enhanced configuration (context roots, TLS, zero downtime deploy, rollback)
+- Support for external load balancers (nginx, haproxy, F5, etc) via extensions
+- Least privilege for extensions (no Docker API access)
+
+Docker Engine running in swarm mode has a routing mesh, which makes it easy to expose your services to the outside world. Since all nodes participate in the routing mesh, users can access a service by contacting any node.  
+
+![swarm routing mess](../images/interlock-overview-1.svg)
+
+For example, a WordPress service is listening on port 8000 of the routing mesh. Even though the service is running on a single node, users can access WordPress using the domain name or IP of any of the nodes that are part of the swarm.
+
+UCP extends this one step further with layer 7 layer routing (also known as application layer 7), allowing users to access Docker services using domain names instead of IP addresses. This functionality is made available through the Interlock component.
```

Comment:
```suggestion
UCP extends this one step further with layer 7 layer routing, allowing users to access Docker services using domain names instead of IP addresses. This functionality is made available through the Interlock component.
```

-  drop the parenthetical